config/samples/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+## Append samples of your project ##
+resources:
+- spanner_v1alpha1_spannerautoscaler.yaml
```

Comment:
```suggestion
- namespace.yaml
- spanner_v1alpha1_spannerautoscaler.yaml
```

Reply:
Thanks for the review, I updated the `kustomization.yaml` as well as the docs to reflect using this through `-k` flag (instead of the old `-f` raw yaml files) as you suggested.

I also saw the `patchesStrategicMerge` deprecation message warning, but changing it (using the recommended `kustomize edit fix` command[^dep-ref]) seemed to be more work and since it is still going to be supported for at least two next versions and until a new major version upgrade, I left it as it is for now. Thank you for offering to help on this though.

[^dep-ref]: https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/patchesstrategicmerge/